ts/features/bonus/cgn/components/CgnTestOverlay.tsx

```diff
@@ -0,0 +1,39 @@
+import { View } from "native-base";
+import * as React from "react";
+import { Platform, StyleSheet } from "react-native";
+
+import { getStatusBarHeight, isIphoneX } from "react-native-iphone-x-helper";
+import { Body } from "../../../../components/core/typography/Body";
+import { Label } from "../../../../components/core/typography/Label";
+import { getAppVersion } from "../../../../utils/appVersion";
+
+const styles = StyleSheet.create({
+  versionContainer: {
+    position: "absolute",
+    top: Platform.select({
+      ios: 20 + (isIphoneX() ? getStatusBarHeight() : 0),
+      android: 0
+    }),
+    left: 0,
+    right: 0,
+    bottom: 0,
+    justifyContent: "flex-start",
+    alignItems: "center",
+    zIndex: 1000
+  },
+  versionText: {
+    padding: 2,
+    backgroundColor: "#ffffffaa"
+  }
+});
+
+/**
+ * Temp overlay created to avoid ambiguity when test cgn versions are released.
+ * @constructor
+ */
+export const CgnTestOverlay: React.FunctionComponent = () => (
```

Comment:
This PR lgtm
What do you think about a refactoring?
Since CGN e BPD use two custom components for the same scope, could it be only one instead?

something like this
`<BetaTestingOverlay title="🛠️ CGN TEST VERSION 🛠️" body="some body"/>`

We should always show the app version so that component should include it as a footer.
`body` prop could be an optional string

```typescript
<View style={styles.versionContainer} pointerEvents="box-none">
    <Label style={styles.versionText}>{`TITLE`}</Label>
    <Body style={styles.versionText}>{"optional body"}</Body>
    <H5 style={styles.versionText}>{`${getAppVersion()}`}</H5>
  </View>
```

![Schermata 2021-03-24 alle 14 54 28](https://user-images.githubusercontent.com/822471/112322074-e344c600-8cb0-11eb-927c-c52876de65ae.png)

cc @fabriziofff